src/utils/server.ts

```diff
@@ -1,22 +1,27 @@
+import * as A from "fp-ts/lib/Array";
 import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/pipeable";
 import os from "os";
 
 export const serverPort = 3000;
 export const serverHostname = "0.0.0.0"; // public
 
 const interestingNetworkInterfaces = new Set(["en0", "eth0"]);
 
-const getIpv4Addresses = (nets: ReadonlyArray<os.NetworkInterfaceInfo>) =>
-  nets.filter(net => net.family === "IPv4" && !net.internal);
+const getIpv4Addresses = (
+  nets: ReadonlyArray<os.NetworkInterfaceInfo>
+): ReadonlyArray<os.NetworkInterfaceInfo> =>
+  nets.filter(net => net.family !== "IPv4" && !net.internal);
```

Comment:
```suggestion
  nets.filter(net => net.family === "IPv4" && !net.internal);
```
![Schermata 2022-01-12 alle 18 04 47](https://user-images.githubusercontent.com/822471/149187475-c7fcdd4e-ad46-4f52-9bbf-08289285b989.png)